pkg/sql/contention/txnidcache/testdata/fifo_cache

```diff
@@ -0,0 +1,123 @@
+# 24240 bytes is enough for a fifoCache with 3 fully filled eviction block.
+init capacity=24240
+----
+cache_size: 0
+
+newInputBlock name=block1 percentageFilled=0.25
+----
+blockSize: 42
+
+insertBlock name=block1
+----
+cache_size: 5056
+
+verifyBlockExistence names=block1
+----
+
+inspectEvictionListShape
+----
+blockIdx: 0, blockSize: 42
+
+ensureEvictionBlockOrdering names=block1
+----
+blockIdx: 0, blockName: block1
+
+newInputBlock name=block2 percentageFilled=0.50
+----
+blockSize: 84
+
+insertBlock name=block2
+----
+cache_size: 11120
+
+verifyBlockExistence names=block1,block2
+----
+
+inspectEvictionListShape
+----
+blockIdx: 0, blockSize: 42
+blockIdx: 1, blockSize: 84
+
+ensureEvictionBlockOrdering names=block1,block2
+----
+blockIdx: 0, blockName: block1
+blockIdx: 1, blockName: block2
+
+newInputBlock name=block3 percentageFilled=1.0
+----
+blockSize: 168
+
+insertBlock name=block3
+----
+cache_size: 19200
+
+verifyBlockExistence names=block1,block2,block3
+----
+
+inspectEvictionListShape
+----
+blockIdx: 0, blockSize: 42
+blockIdx: 1, blockSize: 84
+blockIdx: 2, blockSize: 168
+
+ensureEvictionBlockOrdering names=block1,block2,block3
+----
+blockIdx: 0, blockName: block1
+blockIdx: 1, blockName: block2
+blockIdx: 2, blockName: block3
```

Comment:
I feel like you're missing out on the power of datadriven tests. The point is that you visually verify your expectations when you read this file and you can rewrite it.

It seems like all of `verifyBlockExistence`, `inspectEvictionListShape` and `ensureEvictionBlockOrdering` all should just be something like `show`

```
show
----
block1  42
block2  84
block3  168
```

That tells you the order and the size and the indexes.

Reply:
It also removes the need for `verifyBlockNonExistence`

Reply:
Updated the test so that `show` will output eviction list and a dump of the in-memory map.

Reply:
Hmm, do you think the output of the in-memory map is a bit too verbose ?